### PR TITLE
doc: CLIN-1112 update style documentation

### PR DESCRIPTION
**Pull request by francisl**
Project will link to here instead of duplicating in each project

### CODE REVIEW

File: packages/style/Architecture.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Configuration
+
+### Reference
+
+[Ant Design available theme variable](https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less)
+
+### Main/Global use
+
+First import ant design overide in the main application file
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:50]: missing an 'r' in override ?

File: packages/style/Architecture.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+1. Create a `[Component].module.scss` file in the same directory
+2. Import your style inside your components
+
+```javascript
+import style from '[Component].module.scss';
+// ...
+return () => <div clasName={style.isBeautiful}></div>;
+```
+
+3. To use globaly defined theme variable, load theme from `src/style/theme/default/colors.scss` or `src/style/theme/default/main.scss`
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:68]: variable(s)?

File: packages/style/Architecture.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+```
+
+```
+
+### How to export scss variable to javascript
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:74]: variable(s)?

Comment (evans-g-crsj) [reply packages/style/Architecture.md:74]: Javascript (capitalize everywhere else too)

File: packages/style/Architecture.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
+# Styles and themes
+
+## Strategy
+
+1. Use .less to customize colors and ant design components
+2. Export to scss for the main theme and custom components
+3. Use css module and scss per component
+4. Use scss exported variable for props and inline styling (should be avoid when possible)
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:8]: variable(s)

File: packages/style/Architecture.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# Styles and themes
+
+## Strategy
+
+1. Use .less to customize colors and ant design components
+2. Export to scss for the main theme and custom components
+3. Use css module and scss per component
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:7]: for vs per?

File: packages/style/Architecture.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+4. Use scss exported variable for props and inline styling (should be avoid when possible)
+
+Naming convention for colors variables should follow ant design theme implementation
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:10]: convention(s)

File: packages/style/Architecture.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+### Motivation
+
+- Access to the theme value in JS
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:14]: value(s)

File: packages/style/Architecture.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+- Access of the theme value in css module + scss
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:15]: value(s)

File: packages/style/Architecture.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+        | |-- antd.less
+        | '-- colors.less
+        |-- _colors.scss        :> Ant design colors to import from other scss files or Javascript
+        |-- colors.modules.scss :> Colors classes and variable to be uses inside jsx (className)
+        '-- main.scss      :> General style for the site outside ant design component
+```
+
+everything in legacy `src/theme/` should go in `src/style/theme/default/main.scss` or `src/style/theme/default/colors.scss`
+
+## Configuration
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:42]: configuration(s)

Comment (evans-g-crsj) [reply packages/style/Architecture.md:42]: Reference(s)

File: packages/style/Architecture.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+### Reference
+
+[Ant Design available theme variable](https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less)
+
+### Main/Global use
```
Comment (evans-g-crsj) [inline packages/style/Architecture.md:48]: usage?

**Review by evans-g-crsj — APPROVED** [review]

File: packages/style/Architecture.md (hunk 6 above)
Comment (francisl) [reply packages/style/Architecture.md:10]: c'est `a naming convention` singulier

**Pull request merged**